src/unitxt/llm_as_judge.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List
```

Comment:
shouldn't it be under "metric" dir?

Reply:
No. Unitxt project must be flat (to work without installation. We can add it to the metrics.py file, but I thought it worth new file

Reply:
ok